src/test/resources/features/Returns.feature

```diff
@@ -166,3 +166,8 @@ Feature: Submit ECL Return
       | 10000000   | 113    | 198     | The amount of levy you need to pay for this financial year is £5,424.65.   |
       | 31000000   | 284    | 300     | The amount of levy you need to pay for this financial year is £29,589.04.  |
       | 350000000  | 91     | 256     | The amount of levy you need to pay for this financial year is £175,342.46. |
+
+  Scenario: User wants to go to check your answers page directly without providing any of the previous pages details
+    Given I am signed in to the return journey
+    When I go to the return submission check your answers page directly without providing answers for any of the previous page questions
+    Then I should be on the page that says The answers you provided are not valid
```

Comment:
Is there a scenario/scenarios missing here whereby once on the check your answers page having provided all answers, that they are displayed as expected?

Reply:
This is the unhappy path and the happy path tested in the first scenario